core/src/main/java/org/apache/accumulo/core/client/ClientConfiguration.java

```diff
@@ -221,6 +245,45 @@ public static ClientConfiguration loadDefault() {
     return loadFromSearchPath(getDefaultSearchPath());
   }
 
+  /**
+   * Initializes an empty configuration object to be further configured with other methods on the class.
+   *
+   * @since 1.9.0
+   */
+  public static ClientConfiguration empty() {
```

Comment:
Because of how I have seen this method name used elsewhere, it seems feels like it should return something immutable.  Could call it `from()` or `create()`.

Reply:
I like `create()`